src/alchemlyb/estimators/__init__.py

```diff
@@ -1,3 +1,21 @@
+class EstimatorMixOut():
```

Comment:
Name `_EstimatorMixOut` so that we can keep it internal and don't have to worry about documenting it.